FitGoal/FitGoal/UserProfileConfiguration/ViewControllers/UserProfileConfiguratorViewController.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  UserGenderViewController.swift
+//  FitGoal
+//
+//  Created by Eliany Barrera on 26/4/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import UIKit
+
+class UserProfileConfiguratorViewController: UIViewController {
+    
+    private var configuratorType: UserProfileConfiguratorType
+    
+    private var selectorView: UIView
+    private var questionPrefix = UILabel()
+    private var questionSuffix = UILabel()
+    
+    init(configuratorType: UserProfileConfiguratorType) {
+        self.configuratorType = configuratorType
+        selectorView = configuratorType.view
+        super.init(nibName: nil, bundle: nil)
+        configureQuestions(prefix: configuratorType.prefix, suffix: configuratorType.suffix)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func configureQuestions(prefix: String, suffix: String) {
+        questionPrefix.attributedText = prefix.formattedText(font: "Roboto-Light", size: 19, color: .white, kern: 0.23)
+        questionSuffix.attributedText = suffix.formattedText(font: "Oswald-Medium", size: 50, color: .white, kern: 0.59)
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.isOpaque = false
+        
+        let views = [
+            selectorView,
+            questionPrefix,
+            questionSuffix
+        ]
+        
+        view.addMultipleSubviews(views)
+        setConstraints()
+    }
+    
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        let viewHeight = setHeight(forView: selectorView)
+        selectorView.frame = CGRect(x: 0, y: 0, width: 320, height: viewHeight)
+        selectorView.center = CGPoint(x: view.center.x, y: view.center.y + 30)
+    }
+    
+    private func setHeight(forView view: UIView) -> CGFloat {
```

Comment:
This function name is wrong. It should have the `set` prefix since you aren't setting anything inside it.